### DISCUSSION TITLE
useFetchers with getFetcherType

### DISCUSSION THREAD

**Discussion by theocerutti** [sergiodxa/remix-utils#307] [General]
Hello,

Is it possible to use `useFetchers` with getFetcherType ?

Like

```
const fetchers = useFetchers();

const type = getFetcherType(fetchers[0]);
```

It gives me a typescript error.

Maybe we can modify the typing with:
```
export function useFetcherType(
  fetcher: FetcherWithComponents<unknown> | Fetcher,
)
```

thanks!

**Comment by sergiodxa** [comment]
It should support both, yes, do you want to send a PR?
